src/jdk.compiler/share/classes/com/sun/source/util/DocTreeScanner.java

```diff
@@ -512,8 +512,25 @@ public R visitSnippet(SnippetTree node, P p) {
     /**
      * {@inheritDoc}
      *
+     * @param node {@inheritDoc}
```

Comment:
There's something wrong with the `@implSpec` tags in this and the immediately following two methods. Might be an artifact of the patch.

Reply:
Believed fixed.